extension/packages/nextjs/app/erc20/page.tsx

```diff
@@ -0,0 +1,159 @@
+              <a
+                target="_blank"
+                href="https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol"
+                className="underline font-bold text-nowrap"
+              >
+                ERC-20 token implementation
+              </a>{" "}
+              from the OpenZeppelin library, which is a library for secure smart contract development.
+            </p>
+          </div>
+
+          <div className="divider my-0" />
+          <h2 className="text-3xl font-bold mt-4">Getting Started</h2>
+          <div>
+            <p>
+              <strong>Hardhat: </strong>set your burner wallet address in the deploy script at{" "}
+              <code className="italic bg-base-300 text-base font-bold max-w-full break-words break-all [word-spacing:-0.5rem] inline-block">
+                packages / hardhat / deploy / 01_deploy_se2_token.ts
+              </code>
+              , line 25 (const frontendAddress).
+            </p>
+            <p>
+              <strong>Foundry: </strong>set your burner wallet address in the deploy script at{" "}
+              <code className="italic bg-base-300 text-base font-bold max-w-full break-words break-all [word-spacing:-0.5rem] inline-block">
+                packages / foundry / script / Deploy.s.sol
+              </code>
+              , line 30 (address frontendAddress).
+            </p>
+            <p>
+              Deploy your contract running{" "}
+              <code className="italic bg-base-300 text-base font-bold max-w-full break-words break-all [word-spacing:-0.5rem] inline-block">
+                yarn deploy
+              </code>
+            </p>
+          </div>
+          <div className="divider my-0" />
+
+          <h2 className="text-3xl font-bold mt-4">Interact with the token</h2>
+
+          <div>
+            <p>Below you can see your token balance.</p>
+            <p>Your balance should be 1000 tokens if you set your address before deploying the contract.</p>
+            <p>
+              You can also transfer tokens to another address. Just fill in the address and the amount of tokens you
+              want to send and click the send button.
+            </p>
+            <p>
+              You can test it by opening this page on an incognito window and sending tokens to this new burner wallet
+              address.
+            </p>
+          </div>
+        </div>
+
+        <div className="flex flex-col justify-center items-center bg-base-300 w-full mt-8 px-8 pt-6 pb-12">
+          <div className="flex justify-center items-center space-x-2 flex-col sm:flex-row mb-6">
+            <p className="my-2 mr-2 font-bold text-2xl">Balance:</p>
+            <p className="text-xl">{balance ? formatEther(balance) : 0} tokens</p>
+          </div>
+          <div className="flex flex-col bg-base-100 px-10 py-10 text-center items-center w-2/4 rounded-3xl">
```

Comment:
The form looks a bit funky with the spacing. Let me give it a go!

![image](https://github.com/user-attachments/assets/9a618024-b0d2-4b54-bdb0-a1595e5e21f0)
![image](https://github.com/user-attachments/assets/d770284b-ba31-42fa-8ffe-1f42abee4ce0)